funds1.c

```diff
@@ -0,0 +1,27 @@
+#include <stdio.h>
+#define FUNDLEN 50
```

Comment:
minor nit - I prefer all #define values to have parentheses so we never get in trouble on order of operation when they are used.  
FUNDLN is a weak name as well - FUND_NAME_LENGTH?
```suggestion
#define FUNDLEN (50)
```